src/module_library/daylength_calculator.h

```diff
@@ -0,0 +1,117 @@
+#ifndef DAYLENGTH_CALCULATOR_H
+#define DAYLENGTH_CALCULATOR_H
+
+#include <cmath>
+#include "../framework/module.h"
+#include "../framework/state_map.h"
+
+namespace standardBML
+{
+
+
+/**
+ * @class daylength_calculator
+ *
+ * @brief Computes day_length using a model described by Forsythe et al. (1995).
+ *
+ * Using equation (3) from the paper, the day length \f$D\f$ in hours is
+ *
+ * \f[ D = 24 - \frac{24}{\pi} \cos^{-1} \left( \frac{\sin \rho + \sin \lambda \sin \phi}{ \cos \lambda \cos \phi} \right) \f]
+ *
+ * where \f$\phi = \delta \pi / 180\f$ is the solar declination in radians (computed elsewhere)
```

Comment:
Is this value typically computed by the `BioCro:solar_position_michalsky` module? If so, it would be helpful to mention, so people know how (one way) to calculate it.